packages/web-components/src/components/callout-data/callout-data.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import './callout-data-container';
+import styles from './callout-data.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Callout Data.
+ *
+ * @element dds-callout-data
+ */
+@customElement(`${ddsPrefix}-callout-data`)
+class DDSCalloutData extends LitElement {
+  render() {
+    return html`
+      <dds-callout-data-container class="test">
```

Comment:
Any reason to have `<dds-callout-data-container>` as a separate component, instead of `<div class="${prefix}--callout__column">`?

Reply:
@asudoh I had to style callout deeper than a slotted callout would permit, so creating a new component lilke this allow me to have more control over css.

Reply:
OK thank you for explaining @RaphaelAmadeu! I think the case here seems to be different from the case with `<dds-content-block-heading>`, etc. We have created `<dds-content-block-heading>`, etc. given the element is placed at light DOM where `<dds-content-block>`'s shadow DOM style does no control over. OTOH e.g. if we put `<div class="${prefix}--callout__column">` (or `<section class="${prefix}--callout__container">`) here in shadow DOM, the shadow DOM style of `<dds-callout-data>` has the control over the the `<div>` (or `<section>`). Does this make sense? Feel free to ping me otherwise - Thanks!